libs/nlohmannjson/Makefile

```diff
@@ -34,9 +34,10 @@ endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include/nlohmann
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/nlohmann/json.hpp $(1)/usr/include/nlohmann
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/include/nlohmann/json.hpp $(1)/usr/include/nlohmann
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/include/nlohmann/json_fwd.hpp $(1)/usr/include/nlohmann
```

Comment:
I do have one question about it as I did not look at it more closely. I see you added json_fwd.hpp, could you describe why it was added to the commit description, so we know in the future why it was added?

Reply:
Sure!
From version 3.11.2 on forward template declarations are collected in the file json_fwd.hpp. The json.hpp still contains the template definitions.
Because in most compilation units of a software package only the file json_fwd.hpp needs to be included, the compilation process is accelerated a lot.

I have also added this information to the commit message.